setup.cfg

```diff
@@ -18,9 +18,23 @@ line_length=50
 ############### flake8 ###############
 
 [flake8]
-ignore = 
-    D203
-exclude = 
+ignore =
+    D203,
+    # Ignore missing docstring in public modules
+    D100,
+    # Ignore missing docstring in public packages
+    D104,
+    D202,
+    # Don't force empty line before docstrings
+    D203,
+    # Docstring can be on first line
+    D212,
+    # Doc string in __init__() function should be in class instead
+    D107,
+    D402,
```

Comment:
Could use comment on what these are for.
Example: D402: First line should not be the function’s “signature”